### ISSUE TITLE
Unify scrip_mesh.py and ICON_SCRIP.py

### ISSUE THREAD

**Issue by mvhulten** [HPSCTerrSys/eCLM_static-file-generator#8]
This means, either
- combine in one script, or
- make it work similarly (e.g. both should accept `--ifile`...)

**Comment by mvhulten** [comment]
@s-poll, `ICON_SCRIP.py` contains much less than `script_mesh.py`, e.g. longopts but also whole routines: apparently corner calculation can be done much simpler?

I'll try to use `ICON_SCRIP.py` instead of `scrip_mesh.py` and see what happens.

Depending on results, maybe I should not make big changes in this repo.

I know, the overall structure of the different repos should be streamlined, but have to start somehwer...

**Comment by s-poll** [comment]
I agree and would welcome if the `ifile` would be an option and not hardcoded in the script. The information of edges of the grid can be obtained by the ICON grid file and does not need to be calculated.

**Comment by mvhulten** [comment]
- Combination into one script is not strictly needed an put on a backburner
- `ifile` and such are now used for both scripts
